templates/web/nottinghamshirepolice/report/new/councils_text_all.html

```diff
@@ -0,0 +1,18 @@
+[% category = mark_safe(category) %]
+
+<p>
+[% UNLESS non_public_categories.$category;
+
+    tprintf(
+        loc('These will be sent to the <strong>Immediate Justice Team</strong> and also published online for others to see, in accordance with our <a href="%s">privacy policy</a>.'),
+        c.cobrand.privacy_policy_url
+    );
+
+ELSE;
+
+    loc('These will be sent to the <strong>Immediate Justice Team</strong> but not published online.');
```

Comment:
This needs wider thinking - presumably if it's referred to a council, it will be made public, and there are various places (like here) the site will be saying this won't happen, so that will need careful thinking about.

Reply:
Actually, perhaps this is the only place! In which case, needs changing :)

Reply:
Perhaps "These will be sent to the <strong>Immediate Justice Team</strong>" and leave the question to the privacy statement or elsewhere? Or if it can be put here that it might be public if referred to a council (though presumably the police might moderate contents etc, I dunno).
(Doesn't need a `loc()`, not being translated)